nesta_ds_utils/nesta_io.py

```diff
@@ -0,0 +1,41 @@
+
+from io import BytesIO
+from typing import Union
+import pathlib
+import zipfile
+import logging
+
+logger = logging.getLogger(__name__)
+
+def convert_str_to_pathlib_path(path: Union[pathlib.Path, str]) -> pathlib.Path:
+    """Converts a path written as a string to pathlib format
+
+    :param path: file path in string format
+    :type path: str or pathlib.path
+    :return: path in pathlib format
+    :rtype: pathlib.Path
+    """
+    return pathlib.Path(path) if type(path) is str else path
+
+def make_path_if_not_exist(path: Union[pathlib.Path, str]):
```

Comment:
Stylistic: we mostly follow the [Google Python style guide](https://google.github.io/styleguide/pyguide.html#35-blank-lines) which specifies 2 blank lines between top level definitions. So essentially, for everything except methods defined in classes.